### ISSUE TITLE
Anyone SuccessFul getting an Trigger Alert ? 

### ISSUE THREAD

**Issue by mihirkemkar** [insidesherpa/JPMC-tech-task-3-PY3#25]
Hello Guys, 

Can anyone confirm with me that they have got a triggered alert on data ? I tried, it also depends on incoming data, but if anyone successful, can help me with it ? 

Many thanks

**Comment by tonyksong** [comment]
Yes. Try making sure that you spelled trigger_alert correctly in all places.
